### PR TITLE
DOCS-448: system.settings

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

### CODE REVIEW

**Review by Enmk — APPROVED** [review]

**Review by Enmk — APPROVED** [review]
Looks lovely, wish there was a way to automate this kind of formatting.

File: docs/en/operations/system_tables.md [hunk 1]
```diff
@@ -832,30 +832,53 @@ WHERE
...
--   `value` (String) — Setting value.
--   `description` (String) — Setting description.
--   `type` (String) — Setting type (implementation specific string value).
--   `changed` (UInt8) — Whether the setting was explicitly defined in the config or explicitly changed.
--   `min` (Nullable(String)) — Get minimum allowed value (if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
--   `max` (Nullable(String)) — Get maximum allowed value (if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
--   `readonly` (UInt8) — Can user change this setting (for more info, look into [constraints](settings/constraints_on_settings.md#constraints-on-settings)).
+- `name` ([String](../data_types/string.md)) — Setting name.
+- `value` ([String](../data_types/string.md)) — Setting value.
+- `changed` ([UInt8](../data_types/int_uint.md#uint-ranges)) — Shows whether a setting is changed during the session.
```
Comment (qoega) [inline docs/en/operations/system_tables.md:843]: Changed from default AFAIK. Settings that are defiled in configuration file are also shown in changed.

File: docs/en/operations/system_tables.md [hunk 2]
```diff
@@ -832,30 +832,53 @@ WHERE
...
+- `description` ([String](../data_types/string.md)) — Short setting description. 
+- `min` ([Nullable](../data_types/nullable.md)([String](../data_types/string.md))) — Minimum value of the setting, if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings). If the setting has no minimum value, contains [NULL](../query_language/syntax.md#null-literal). 
+- `max` ([Nullable](../data_types/nullable.md)([String](../data_types/string.md))) — Maximum value of the setting, if any is set via [constraints](settings/constraints_on_settings.md#constraints-on-settings). If the setting has no maximum value, contains [NULL](../query_language/syntax.md#null-literal). 
+- `readonly` ([UInt8](../data_types/int_uint.md#uint-ranges)) — Shows whether the current user can change the setting:
```
Comment (qoega) [inline docs/en/operations/system_tables.md:847]: In previous version there is a link to find information why this setting is readony. In new one it is lost
`(for more info, look into [constraints](settings/constraints_on_settings.md#constraints-on-settings))`

File: docs/en/operations/system_tables.md [hunk 3]
```diff
@@ -832,30 +832,53 @@ WHERE
...
+
+```sql
+SELECT
+    name,
+    value,
+    changed,
+    description,
+    min,
+    max,
+    readonly
```
Comment (qoega) [inline docs/en/operations/system_tables.md:862]: If all columns are selected why not use *

File: docs/en/operations/system_tables.md [hunk 4]
```diff
@@ -832,30 +832,53 @@ WHERE
...
 FROM system.settings
-WHERE changed
```
Comment (qoega) [inline docs/en/operations/system_tables.md:856]: Example with WHERE changed is the top scenario for a user. 
- Check that his chahges in xml are used and read correctly. 
- Check whether she already set something in current session.
- What configuration is set on a cluster

It is better not to lose this case.

**Review by qoega — CHANGES_REQUESTED** [review]
Currently not OK

File: docs/en/operations/system_tables.md (hunk 2 above)
Comment (BayoNet) [reply docs/en/operations/system_tables.md:852]: That's not about constraints, there are complicated set of conditions which lead to readonly. There is no proper text in docs.

**Pull request merged**